src/main/java/com/github/dockerjava/api/command/BuildImageCmd.java

```diff
@@ -87,6 +88,9 @@
     @CheckForNull
     public String getCpusetcpus();
 
+    @CheckForNull
+    public Map<String, String> getBuildArgs();
```

Comment:
Could you add javadoc? Example:
https://github.com/docker-java/docker-java/blob/9f779727004c4f87e2b4426145e3b1bdb302b39f/src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java#L360-L364
reference to field
https://github.com/docker-java/docker-java/blob/9f779727004c4f87e2b4426145e3b1bdb302b39f/src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java#L278-L281